docs/src/main/asciidoc/native-and-ssl.adoc

```diff
@@ -206,7 +206,7 @@ You can for example modify your `Dockerfile.native` as follows to copy the requi
 ==== Java 8
 [source, subs=attributes+]
 ----
-FROM quay.io/quarkus/ubi-quarkus-native-image:{graalvm-version}-java8 as nativebuilder
+FROM quay.io/quarkus/ubi-quarkus-native-image:{graalvm-flavor} as nativebuilder
```

Comment:
This doc section is about Java 8 but the default `{graalvm-flavor}` is `...-java11`. Is that OK to keep things that way?